engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/confluent/ConfluentSchemaRegistryClient.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.engine.avro.confluent
+
+import io.confluent.kafka.schemaregistry.client.{CachedSchemaRegistryClient, SchemaRegistryClient => ConfluentKafkaSchemaRegistryClient}
+import io.confluent.kafka.serializers.KafkaAvroDeserializerConfig
+import org.apache.avro.Schema
+import pl.touk.nussknacker.engine.avro.SchemaRegistryClient
+import pl.touk.nussknacker.engine.kafka.KafkaConfig
+
+class ConfluentSchemaRegistryClient(var confluentClient: ConfluentKafkaSchemaRegistryClient) extends SchemaRegistryClient with Serializable {
```

Comment:
Why is it var? And how can this class be serializable if CachedSchemaRegistryClient is not??

Reply:
I've removed it. Now we have ConfluentSchemaRegistryClientFactory which is serializable.